src/main/java/org/cyclonedx/model/Evidence.java

```diff
@@ -35,6 +36,7 @@
 public class Evidence extends ExtensibleElement {
 
     private LicenseChoice license;
+    @JacksonXmlElementWrapper(localName = "copyright")
```

Comment:
Try `    @JacksonXmlElementWrapper(useWrapping = false)`, I think that's what you are after